range_queries/prefix_sum_array.cpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * @file prefix_sum_array.cpp
+ * @brief Implementation of [Prefix Sum Array] data structure
+ *
+ * @details
+ * Prefix Sum Array is a data structure, that allows answering sum in some range
+ * queries. It can answer most sum range queries in O(1), with a build time
+ * complexity of O(N). But it hasn't an update querie.
+ *
+ * * Running Time Complexity \n
+ * * Build : O(N) \n
+ * * Range Query : O(1) \n
+ */
+
+#include <cassert>
+#include <iostream>
+#include <vector>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```